Ragnarok/Bragi/Bragi.h

```diff
@@ -21,14 +22,21 @@
 	#include <tchar.h>
 	#include <Userenv.h>
 	#include <shlobj.h>
+	#include <wchar.h>
+	#include <sys/stat.h>
+	#include <sys/types.h>
+	#include <Shlwapi.h>
 
-	/*#pragma comment(lib, "User32.lib")*/
+	#pragma comment(lib, "Shlwapi.lib")
 	#define LEN 64
-	#define PATH 128
+	#define PATH 512
```

Comment:
In my opinion, the max path size should be at least 2048 in case there are too many subfolders